integrations/anthropic/tests/test_chat_generator.py

```diff
@@ -481,15 +613,19 @@ def test_convert_streaming_chunks_to_chat_message(self):
                     "delta": {"stop_reason": "tool_use", "stop_sequence": None},
                     "usage": {"completion_tokens": 40},
                 },
+                component_info=ComponentInfo.from_component(self),
+                index=1,
```

Comment:
So I'd say this streaming chunk is also not part of the content block since it's of type `message_delta` instead of `content_block_delta`. So I'd also assign the index to None here.